src/System.Threading.Tasks.Extensions/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0008</DiagnosticId>
+    <Target>T:System.Runtime.CompilerServices.AsyncMethodBuilderAttribute</Target>
```

Comment:
Do we understand the reason for this compatibility warning?  What interface is it complaining was removed?

Reply:
Running this locally I see:
```
error CP0008: Type 'System.Runtime.CompilerServices.AsyncMethodBuilderAttribute' does
not implement interface 'System.Runtime.InteropServices._Attribute' on lib/netstandard2.0/System.Threading.Tasks.Extensions.dll but it does on [Baseline] lib/net461/System.Threading.Tasks.Extensions.dll
```

Should we be excluding `net461` from the baseline?   If we do I think this won't show up.  It's happening because we're comparing the NETFramework assembly -- where the base `Attribute` class implements `_Attribute` but the netstandard2.0 assembly does not.

Reply:
Can you please help me understand this? How would re-adding net461 "fix" the warning? We are still not implementing _Attribute in newer tfms.